astro/src/components/api/AvailableSince.astro

```diff
@@ -0,0 +1,4 @@
+---
+const { since } = Astro.props;
+---
+{ since && <span class="px-3 text-green-500 italic capitalize">available since {since}</span> }
```

Comment:
Just a thought here but unlike in APIField there isn't a good reason for `since` to be null on this component. We may want to instead of doing a null check throw an error out of the frontmatter if the prop is null, thus ensuring that all usages have the proper input. Should throw errors at build time.